BookClub/templates/review_details.html

```diff
@@ -87,7 +87,7 @@
                                     <span>Edit</span>
                                 </button>
                             </a>
-                            {% include 'partials/delete_button_and_modal.html' with delete_item=review delete_modal_id=review.pk %}
+                            {% include 'partials/delete_button_and_modal.html' with delete_item=review delete_modal_id=0 %}
```

Comment:
why zero here?

Reply:
Because usually i pass the pk, but since we may potentially need a delete for 2 different models (review and comments), we can't guarantee their pks are never same (both can be 1 for example) (causes wrong modals linked to buttons). However, we know that pk is never 0, and in a ReviewDetailView we only display one object of Review model, so it seemed like an easy and straightforward way to guarantee avoiding pk collisions

Reply:
ah okay. If you've manually tested the modals for both then I'm happy with that.